src/types/posters.ts

```diff
@@ -1,15 +1,10 @@
 export type Poster = {
-  authorName: string;
-  categories: string[];
-  title: string;
-  authors?: string[];
-  contactEmail?: string;
+  description?: string;
+  descriptionSecondary?: string;
```

Comment:
Can be `title`, `subtitle`, `description` to keep it similar to other places in code

Reply:
This is what we've agreed with the client. Spreadsheets have already started being built based on these names :(